src/main/java/net/geforcemods/securitycraft/SCTags.java

```diff
@@ -14,11 +14,13 @@
 		public static final Tag<Block> REINFORCED_ACACIA_LOGS = tag("reinforced/acacia_logs");
 		public static final Tag<Block> REINFORCED_BIRCH_LOGS = tag("reinforced/birch_logs");
 		public static final Tag<Block> REINFORCED_CARPETS = tag("reinforced/carpets");
+		public static final Tag<Block> REINFORCED_COBBLESTONES = tag("reinforced/cobblestones");

```

Comment:
To be in line with Tags.Blocks.COBBLESTONE:
```suggestion
		public static final Tag<Block> REINFORCED_COBBLESTONE = tag("reinforced/cobblestone");
```